src/Symfony/Bundle/SecurityBundle/composer.json

```diff
@@ -17,7 +17,7 @@
     ],
     "require": {
         "php": ">=5.5.9",
-        "symfony/security": "~3.1,>=3.1.2",
+        "symfony/security": "~3.2",
```

Comment:
Can be removed - on the `3.4` branch, this is already updated to `"symfony/security": "~3.4|~4.0",`